Makefile

```diff
@@ -28,7 +28,7 @@ GOBIN=$(shell pwd)/bin
 
 .PHONY: bin/aws-ebs-csi-driver
 bin/aws-ebs-csi-driver: | bin
-	CGO_ENABLED=0 GOOS=linux go build -ldflags ${LDFLAGS} -o bin/aws-ebs-csi-driver ./cmd/
+	CGO_ENABLED=0 GOOS=linux go build -mod=vendor -ldflags ${LDFLAGS} -o bin/aws-ebs-csi-driver ./cmd/
```

Comment:
nit: use GOFLAGS instead?

Reply:
if it is all the same, I would like to keep this to explicit flag.